src/components/editMeasure/editor/StatusHandler.tsx

```diff
@@ -161,9 +159,7 @@ const StatusHandler = ({
     return <></>;
   }
 
-  return (
-    <MadieAlert alerts={alerts} minimizeAlerts={featureFlags?.MinimizeAlerts} />
-  );
+  return <MadieAlert alerts={alerts} minimizeAlerts={true} />;
```

Comment:
Same idea as https://github.com/MeasureAuthoringTool/madie-cql-library/pull/237/files#r2310968415, can the prop be removed and defaulted?